packages/fxa-auth-server/test/remote/account_create_tests.js

```diff
@@ -409,13 +413,55 @@ describe('remote account create', function() {
       .then(assert.fail, err => assert.equal(err.errno, 107));
   });
 
+  it('invalid entrypointExperiment', () => {
```

Comment:
I realize the new fields are marked optional, should we assert that missing entrypointExperiment/Variation metrics are still accepted? This would ensure that we don't start rejecting metrics during the deployment since the auth server goes out before the content server.